### PR TITLE
Release notes for 2021.1.1 Eos

### PR DESCRIPTION
**Pull request by digisomni**
Will update the .rpm binary when that's sorted, something's wrong with the build process atm.

### CODE REVIEW

File: docs/source/release-notes/2021-1-1-Eos.md [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+
+The numbers at the end of each item are the PR numbers in the Vircadia [repo](https://github.com/vircadia/vircadia).
+
+#### General
+
+* UI/UX: Add an onboarding wizard for first run configuration of Vircadia. (#999)
+* UI/UX: Add a manual noise reduction mode with a slider for adjustment. (#1007, #1097)
+* API: `Controller.triggerHapticPulse` now supports selecting an arbitrary device instead of triggering two hands every time. (#1013)
+* Application: Non-uniformly scaled sphere zones and models now behave like ellipsoids. (#1033)
+* API: Fix `keepAspectRatio` property of Image Entities to correctly scale with any ratio. (#1050, #1141)
```
Comment (ctrlaltdavid) [inline docs/source/release-notes/2021-1-1-Eos.md:15]:
```suggestion
* API: Fix `keepAspectRatio` property of Image entities to correctly scale with any ratio. (#1050, #1141)
```

File: docs/source/release-notes/2021-1-1-Eos.md [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+* Graphics: Fix glTF vertex colors. (#1123)
+* Graphics: Fix parts of avatars rotating incorrectly. (#1124)
+
+#### Miscellaneous
+
+* Docs: Update build instructions for the Oculus Quest platform. (#800)
+* Docs: Add full packaging instructions for Linux-based servers. (#899)
+* Docs: Document addition of the `RELEASE_NAME` environment variable for versions. (#1014)
+* Application: Add names to some threads to aid in debugging. (#1040)
+* Docs: Remove nvtt workaround from the Linux build documentation.
```
Comment (ctrlaltdavid) [inline docs/source/release-notes/2021-1-1-Eos.md:38]: PR?

File: docs/source/release-notes/2021-1-1-Eos.md [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+* Build: Update Linux-based server packaging scripts to allow input of versions. (#1055)
+* Build: Add the release name to the end of installers' names. (#1057)
+* Application: Set resource total size to content size when using a `qrc://` resource. (#1068)
+* Application: Fix 'General' settings privacy option text to correctly reference 'Vircadia'. (#1071)
+* Docs: Define code blocks to fix Sphinx warnings in Vircadia docs. (#1078, #1083)
+* Build: Add the ability to preload an entity script whitelist when building. (#1081)
+* Application: Implement machine fingerprints for Linux. (#1092)
+* Misc: Add 'Stale' bot configuration to the repository. (#1117)
+* Application: Fix `VIRCADIA_MEMORY_DEBUGGING` on Linux/gcc. (#1127)
+* Application: Fix a stack-use-after-return error in the `RenderableModelEntityItem` class.
```
Comment (ctrlaltdavid) [inline docs/source/release-notes/2021-1-1-Eos.md:49]: PR?

**Review by ctrlaltdavid — CHANGES_REQUESTED** [review]

File: docs/source/release-notes/2021-1-1-Eos.md (hunk 2 above)
Comment (JulianGro) [reply docs/source/release-notes/2021-1-1-Eos.md:38]: https://github.com/vircadia/vircadia/pull/1044

File: docs/source/release-notes/2021-1-1-Eos.md (hunk 3 above)
Comment (JulianGro) [reply docs/source/release-notes/2021-1-1-Eos.md:49]: Probably this https://github.com/vircadia/vircadia/pull/1132

**Review by ctrlaltdavid — APPROVED** [review]

**Pull request merged**